project/ProjectPlugin.scala

```diff
@@ -103,9 +109,9 @@ object ProjectPlugin extends AutoPlugin {
       orgSupportedScalaJSVersion := Some("0.6.15"),
       orgScriptTaskListSetting ++= List(
         (ScoverageKeys.coverageAggregate in Test).asRunnableItemFull,
-       "docs/tut".asRunnableItem
+        "docs/tut".asRunnableItem
       ),
-      coverageExcludedPackages := "<empty>;github4s\\.scalaz\\..*",
+      coverageExcludedPackages := "<empty>;github4s\\.scalaz\\..*;github4s\\.cats\\.effect\\..*",
```

Comment:
Just thought I'd exclude them like what is being done for scalaz, happy to include them.